tests/e2e/httpd.spec.ts

```diff
@@ -10,17 +10,13 @@ test("rad web command reacts to port change", async ({ page, peerManager }) => {
   await page.getByRole("button", { name: "radicle.local" }).click();
 
   await expect(
-    page.getByText(
-      "rad web --frontend http://localhost:3001 --backend http://127.0.0.1:8081",
-    ),
+    page.getByText("rad web --backend http://127.0.0.1:8081"),
   ).toBeVisible();
   await page.locator('input[name="port"]').fill("8090");
   await page.locator('input[name="port"]').press("Enter");
 
   await expect(
-    page.getByText(
-      "rad web --frontend http://localhost:3001 --backend http://127.0.0.1:8090",
-    ),
```

Comment:
This change is due to the app now running in production mode, which by code, doesn't show a different `--frontend` option